### PR TITLE
Changed explode to true

### PR DESCRIPTION
**Pull request by dgarcia360**
Related https://github.com/nemtech/catapult-rest/issues/270

### CODE REVIEW

**Comment by Vektrat** [comment]
aren't we going with the `deepObject` option?

if so, we need to turn `style` to `deepObject` as well

**Comment by dgarcia360** [comment]
@Vektrat Fernando didn't confirm if deepObject (``?type[]=16974&type[]=16718``) is compatible with current OpenAPI generators. In [his last message](https://github.com/nemtech/catapult-rest/issues/270#issuecomment-595416031), he suggested using explode:true (``?type=16974&type=16718 ``) instead.

**Comment by fboucquez** [comment]
I think the default open API option form exploded is the best middle ground, 

?type=16724&type=16705&type=17220

This works in java and ts without custom templates

I've just tried the deepObject style and java generated clients don't support it. Could we go with the form exploded @dgarcia360 @Vektrat @rg911?

I wouldn't use deep object query param anyway, if we want to serialize complex objects, I would send a post JSON body.

https://github.com/OAI/OpenAPI-Specification/issues/1706

**Comment by dgarcia360** [comment]
I'd go with `explode: true` too, waiting for confirmation from @Vektrat and @martinayora to merge this PR.

**Comment by Vektrat** [comment]
i'm unsure if it's directly supported by the REST api so this will need to wait from our side until i try it

File: spec/parameters/transactionTypes.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: type
 in: query
 description: Transaction types to filter by, separated by commas.
```
Comment (fboucquez) [inline spec/parameters/transactionTypes.yml]: The description would need to be changed. A query example could be shown

**Pull request merged**